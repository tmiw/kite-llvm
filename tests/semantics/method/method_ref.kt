method x(a) [ a|print; ];
method x(a, b) [ (a+b)|print; ];
method x() [ 31337|print; ];

v = method_ref(this, "x", 0);
v();
v = method_ref(this, "x", 1);
v(1);
v = method_ref(this, "x", 2);
v(1, 2);