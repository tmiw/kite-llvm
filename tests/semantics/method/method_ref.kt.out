31337
1
3
